File: 프로그래머스/0/120887. k의 개수/k의 개수.java
class Solution {
    public int solution(int i, int j, int k) {
        int count = 0;
        char kChar = (char) (k + '0');
        
        for (int a = i; a <= j; a++) {
            String b = String.valueOf(a);
            for (int m = 0; m < b.length(); m++) {
                if (b.charAt(m) == kChar) {
                    count++;
                }
            }
        }
        return count;
    }
}

File: 프로그래머스/0/120887. k의 개수/k의 개수.cs
using System;

public class Solution {
    public int solution(int i, int j, int k) {
        int count = 0;
        char kChar = (char)(k + '0');
        
        for (int a = i; a <= j; a++) {
            string b = a.ToString();
            for (int m = 0; m < b.Length; m++) {
                if (b[m] == kChar) {
                    count++;
                }
            }
        }
        return count;
    }
}
